backend/utils/service-utils.js

```diff
@@ -37,5 +37,35 @@ module.exports = {
         if (!price) return false;
         else if (typeof price != "number") return false;
         else return true;
-    }
+    },
+    dynamicallyValidateDuration: function dynamicallyValidateDuration(duration) {
+        if (!duration) {
+            return true;
+        }
+        else {
+            let isValid = durationFormat.test(duration);
+            if(!isValid) {
+                return false
+            }
+            else {
+                let hours = parseInt(duration.slice(0, 2));
+                let minutes = parseInt(duration.slice(3, 5));
+                if((hours > 8) || (hours == 8 && minutes > 0)) {
```

Comment:
Please, use the strict comparison operator (===) instead of abstract (==).
See https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript